.github/workflows/dotnet_nugets.yml

```diff
@@ -6,6 +6,8 @@ name: .NET Nugets
 on:
   workflow_dispatch:
   push:
+    tags:
+      - '*'
```

Comment:
The VW version checks that the tag looks like a SemVer number. Do you think this is also good to have here?
```
    tags:
      - '[0-9]+.[0-9]+.[0-9]+'
      - '[0-9]+.[0-9]+.[0-9]+-*'
```

Reply:
I don't think we expect to use different tags, so it doesn't feel necessary to me